python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -405,6 +406,15 @@ def run_transformer_int4_gpu(repo_id,
 
     model = BenchmarkWrapper(model)
 
+    csv_data = []
+    try:
+        with open(csv_name, 'r') as csvfile:
+            csv_reader = csv.reader(csvfile)
+            csv_data = [row for row in csv_reader]
```

Comment:
why do we need to read the entire file? We just need to write?

Reply:
Have modified it.